source/utilities.tex

```diff
@@ -5170,12 +5170,18 @@
 
 \indexlibrary{\idxcode{any}!constructor}%
 \begin{itemdecl}
-template <class T, class... Args>
-  explicit any(in_place_type_t<T>, Args&&... args);
+template <class ValueType, class... Args>
+  explicit any(in_place_type_t<ValueType>, Args&&... args);
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
+Let \tcode{T} be equal to \tcode{decay_t<ValueType>}.
```

Comment:
It's NOT in 2754 - you must be looking at the superceded versions.  See http://wg21.cmeerw.net/lwg/issue2754 (or skip past the superceded versions in http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/p0165r3.html).